ubuntu/gpu/python/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM databricksruntime/gpu-base:cuda10.1
```

Comment:
Were you able to build this image? Right now we tag them `latest` for lack of a better tagging/versioning system

Reply:
I tagged it locally to build this image. For GPU containers, the CUDA version is the most important one.